src/views/preview/preview.scss

```diff
@@ -403,8 +404,12 @@ $stage-width: 480px;
             width: 100%;
         }
 
-        > .description-block:first-child {
-            margin-top: 1rem;
+        > * {
```

Comment:
I think what you've got is probably the best solution. I just wanted to point out that the original `first-child` css was to make sure the top of the Instructions lined up with the top of the stage where there were no credits.
![Screen Shot 2019-05-20 at 2 57 37 PM](https://user-images.githubusercontent.com/399209/58045241-ddf41e80-7b0f-11e9-9496-6469e7a58cbf.png)

Reply:
@carljbowman just want to confirm that this is ok for the case where there are no remix credits.

Reply:
That seems fine with me, thanks for checking. It was nice that those lined up but not critical. 

Chatted with @benjiwheeler quickly on a follow-up question: the player adds a `0.5rem`  padding-top to the of the controls. This padding makes sense for full screen, but in the project page player view could we drop the 0.5rem padding to get some more clean alignment:

![2019-05-20 15 56 07](https://user-images.githubusercontent.com/3409578/58048238-47782b00-7b18-11e9-9c78-1de0591adeb0.gif)

Reply:
The only problem with @carljbowman's idea is that it involves a change to scratch-gui's css, for a problem in www.

Here's what I came up with:

Add a .25rem padding-top to the div that wraps the instructions/remix credit column. That way the top item is brought into line with the top of the fullscreen button. 

Result:

![image](https://user-images.githubusercontent.com/3431616/58068220-7b763f00-7b5e-11e9-9e5b-970a788117d0.png)

![image](https://user-images.githubusercontent.com/3431616/58068213-74e7c780-7b5e-11e9-8e1a-bff14a188324.png)

Reply:
@benjiwheeler - While the addtional .25rem padding-top aligns "Instructions" the fullscreen button, it pushes that entire column and the bottom of the player and the bottom of "Notes and Credits" become misaligned. 

![misaglinement](https://user-images.githubusercontent.com/3409578/58096301-c0bc6000-7ba2-11e9-97d0-f7cb60b2f1df.png)

If we had to choose, I'd prefer to keep the bottoms of the player and "Notes and Credits" aligned. They have much stronger horizontal lines for the eye to follow, so the misalignment feels much more apparent.

Reply:
Changing the stage css (that's what this would require) is tricky. It's already handling normal, small stage and fullscreen. FWIW, I would just align the bottoms.

Reply:
I agree, I am aligning the bottoms.